src/vs/workbench/parts/debug/electron-browser/debugEditorContribution.ts

```diff
@@ -192,6 +200,42 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 		});
 	}
 
+	private onFocusStackFrame(stackFrame: IStackFrame): void {
+		// Since step over, step out is a fast continue + break. Continue clears stack.
+		// This means we'll get a null stackFrame followed quickly by a valid stackFrame.
+		// Removing all decorators and adding them again causes a noticeable UI flash due to relayout and paint.
+		// We want to only remove inline decorations if a null stackFrame isn't followed by a valid stackFrame in a short interval.
+		clearTimeout(this.removeDecorationsTimeoutId);
```

Comment:
I understand that but this is still not a nice solution. Though it is fine for the first version and once we merge this in next week I can try to polish this if possible